src/exclusive-or.rst

```diff
@@ -375,28 +373,27 @@ Remaining problems
 Real one-time pads, implemented properly, have an extremely strong
 security guarantee. It would appear, then, that cryptography is over:
 encryption is a solved problem, and we can all go home. Obviously,
-that's not the case.
-
-One-time pads are rarely used, because they are horribly impractical:
-the key is at least as large as all information you'd like to transmit,
-*put together*. Plus, you'd have to exchange those keys securely, ahead
-of time, with all people you'd like to communicate with. We'd like to
-communicate securely with everyone on the Internet, and that's a very
-large number of people. Furthermore, since the keys have to consist of
-truly random data for its security property to hold, key generation is
-fairly difficult and time-consuming without specialized hardware.
-
-One-time pads pose a trade-off. It's an algorithm with a solid
-information-theoretic security guarantee, which you can not get from any
-other system. On the other hand, it also has extremely impractical key
-exchange requirements. However, as we'll see throughout this book,
-secure symmetric encryption algorithms aren't the pain point of modern
-cryptosystems. Cryptographers have designed plenty of those, while
-practical key management remains one of the toughest challenges facing
-modern cryptography. One-time pads may solve a problem, but it's the
+that is not the case.
+
+One-time pads are rarely used for being horribly impractical.
+The key is at least as large as all information you would like transmitted,
+*put together*. Plus, the keys must be exchanged securely, ahead
+of time, with all people in your communication network. We like to
```

Comment:
"in your communication network" seems to change meaning for me by getting rid of strangers communicating